cfgov/search/models/__init__.py

```diff
@@ -0,0 +1 @@
+from .models import Synonym
```

Comment:
Views may differ here, but for the tiny `search` app, it seems overkill to have models.py file in a models folder with an __init__ import.

Unless we foresee a big future for search models, I'd vote for simplicity and a single models.py file at /cfgov/search/

Reply:
I can move it